planner/core/task.go

```diff
@@ -525,12 +522,10 @@ func (p *PhysicalHashJoin) attach2Task(tasks ...task) task {
 	lTask := finishCopTask(p.ctx, tasks[0].copy())
 	rTask := finishCopTask(p.ctx, tasks[1].copy())
 	p.SetChildren(lTask.plan(), rTask.plan())
-	p.schema = BuildPhysicalJoinSchema(p.JoinType, p)
```

Comment:
Why remove this line?

Reply:
Hash Join should not call `BuildPhysicalJoinSchema`, this line is added by this pr, but it's unnesessary. Hash join's schema should keep the same no matter which one is small table.